paulmikulskis/redditat
# spamcntrl-landing page - query scans with email as filter

test:
1. `git checkout` and `git 42-spamcntrl-landingpage-mystats-scans-query-filter-by-email`
2. run spamcntrl-landingpage
3. go to `localhost:3000/mystats`
4. try changing the email to your email: https://prnt.sc/L2_Mn-_vWfJP
5. should see dummy data from /scans on the /mystats page reflected.
